lang.go

```diff
@@ -291,6 +294,8 @@ func LangToStringShort(lang Lang) string {
 		return "cs"
 	case Cmn:
 		return "zh" // No iso 639-1, but http://www.loc.gov/standards/iso639-2/faq.html#24
+	case Yue:
```

Comment:
The languages are in order alphabetically (in English). These lines should go above `case Zul:`.